instagram/models.py

```
@@ -183,14 +223,18 @@ def update_user_following(self):
         UserFollowing.objects.bulk_create(user_following_list)
         user_following_update_profile_pictures.delay(self.instagram_id)
 
-    def save_from_url_to_file_field(self, field_name: str, file_format: str, file_url: str) -> None:
+    def save_from_url_to_file_field(self, field_name: str, file_format: str, file_url: str, save: bool = True) -> None:
         response = requests.get(file_url, timeout=30)
 
         if not response.status_code == status.HTTP_200_OK:
```

Ruff (0.8.2)

Use `response.status_code != status.HTTP_200_OK` instead of `not response.status_code == status.HTTP_200_OK`

Replace with `!=` operator

(SIM201)